### PR TITLE
Fix url for old versions

### PR DESCRIPTION
**Pull request by kay0u**
## Problem

- *When a new version of Jellyfin is available, the url of the previous version is no longer valid.*

## Solution

- *Change the url if needed*
- *Use dpkg to remove a linter's warning*
- *Bump ldap version*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by kay0u** [comment]
!testme

File: scripts/_common.sh [hunk 1]
```diff
@@ -9,13 +9,17 @@ pkg_version="10.7.7-1"
...
 ffmpeg_pkg_version="4.3.2-1"
-ldap_pkg_version="12.0.0.0"
+ldap_pkg_version="14.0.0.0"
 
 architecture=$(dpkg --print-architecture)
 
 discovery_service_port=1900
 discovery_client_port=7359
 
+ffmpeg_deps="libass9 libbluray2 libc6 libdrm-intel1 libdrm2 libfontconfig1 libfreetype6 libfribidi0 libgcc1 libgmp10 libgnutls30 libmp3lame0 libopus0 libstdc++6 libtheora0 libvdpau1 libvorbis0a libvorbisenc2 libvpx5 libwebp6 libwebpmux3 libx11-6 libx264-155 libx265-165 libzvbi0 libopencl1 zlib1g"
```
Comment (tituspijean) [inline scripts/_common.sh:19]: What I was worried about happened on my RPi4 (`aarch64`): `Package libdrm-intel1 is not available, but is referred to by another package.`

**Comment by tituspijean** [comment]
!gogogadgetotitus

**Comment by tituspijean** [comment]
Thanks @kay0u !

**Pull request merged**